### PR TITLE
Added Dutch language

### PR DESCRIPTION
**Pull request by ErikSchierboom**
This PR adds support for the Dutch language.

### CODE REVIEW

**Comment by ploeh** [comment]
Thank you for your interest in contributing to Numsense. FTR, I'm currently travelling, and have only limited time to evaluate pull requests. In due time, I'll work through my backlog, but it may take weeks :flushed: :sweat:

**Comment by ErikSchierboom** [comment]
No problem, enjoy your travels!

File: Numsense/ObjectOriented.fs [hunk 1]
```diff
@@ -32,7 +32,14 @@ type PolishNumeralConverter () =
         member this.TryParse (s, result) =
             Helper.tryParse Numeral.tryParsePolish (s, &result)
 
+type DutchNumeralConverter () =
```
Comment (ploeh) [inline Numsense/ObjectOriented.fs]: If I delete this, the code still compiles, and all tests pass.

File: Numsense/ObjectOriented.fs [hunk 2]
```diff
@@ -32,7 +32,14 @@ type PolishNumeralConverter () =
...
+        member this.TryParse (s, result) =
+            Helper.tryParse Numeral.tryParseDutch (s, &result)
+
 type Numeral private () =
     static member val English = EnglishNumeralConverter () :> INumeralConverter
     static member val Danish  =  DanishNumeralConverter () :> INumeralConverter
-    static member val Polish  =  PolishNumeralConverter () :> INumeralConverter
\ No newline at end of file
+    static member val Polish  =  PolishNumeralConverter () :> INumeralConverter
+    static member val Dutch   =   DutchNumeralConverter () :> INumeralConverter
```
Comment (ploeh) [inline Numsense/ObjectOriented.fs]: If I delete this, the code still compiles, and all tests pass.

**Comment by ErikSchierboom** [comment]
@ploeh Ah, I forgot to add the tests in the Numsense.UnitTests.CSharp project. The PR has been updated.

**Comment by ploeh** [comment]
Thank you for your interest in contributing to Numsense! It builds and runs all tests without warnings on my machine, so from a technical perspective I think we're good to go :+1:

As I don't read Dutch, I'd like someone who does to review that part of it, if at all possible. I don't expect any errors, but it's always good with a second pair of eyes :smile:

**Comment by ploeh** [comment]
I solicited a review on Twitter: https://twitter.com/ploeh/status/691263301146263552

Please retweet and spread the word :smile:

**Comment by wcabus** [comment]
Verified, don't see any errors in the Dutch parts.

**Comment by Mavamaarten** [comment]
Agreed, the Dutch parts look fine to me.

**Comment by ploeh** [comment]
Thank you, all, for your contribution. It's now live as [Numsense 0.3.0](https://www.nuget.org/packages/Numsense/0.3.0).

**Comment by ErikSchierboom** [comment]
Awesome!

File: Numsense.UnitTests/DutchExamples.fs [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+[<InlineData(       520, "vijfhonderdtwintig")>]
+[<InlineData(       666, "zeshonderdzesenzestig")>]
+[<InlineData(       798, "zevenhonderdachtennegentig")>]
+[<InlineData(       857, "achthonderdzevenenvijftig")>]
+[<InlineData(       999, "negenhonderdnegenennegentig")>]
+[<InlineData(      1000, "duizend")>]
+[<InlineData(      1001, "duizend een")>]
+[<InlineData(      1010, "duizend tien")>]
+[<InlineData(      1066, "duizend zesenzestig")>]
+[<InlineData(      1337, "duizend driehonderdzevenendertig")>]
```
Comment (toolslive) [inline Numsense.UnitTests/DutchExamples.fs]: "dertienhonderdzevenendertig" is also a valid variation for 1337

Comment (mexx) [inline Numsense.UnitTests/DutchExamples.fs]: For a year, right?

Comment (toolslive) [inline Numsense.UnitTests/DutchExamples.fs]: Yes, but not limited to years. You can use for amounts of money, or heights, .... Basically: anywhere where you can write "1337" you can write "dertienhonderdzevenendertig". (btw: Yes, I'm a native speaker)

File: Numsense.UnitTests/DutchExamples.fs [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+[<InlineData(                                          "achthonderd en zeven",        807)>]
+[<InlineData(                                    "negenhonderdnegenenvijftig",        959)>]
+[<InlineData(                                                       "duizend",       1000)>]
+[<InlineData(                                                   "duizend een",       1001)>]
+[<InlineData(                                                   "duizend één",       1001)>]
+[<InlineData(                                                "duizend en een",       1001)>]
+[<InlineData(                                                "duizend en één",       1001)>]
+[<InlineData(                                          "duizend vierenzestig",       1064)>]
+[<InlineData(                                                 "twaalfhonderd",       1200)>]
+[<InlineData(                                 "zestienhonderdzevenenzeventig",       1677)>]
```
Comment (ErikSchierboom) [inline Numsense.UnitTests/DutchExamples.fs]: @toolslive Yes, it is indeed a valid alternative. However, this is the formal writing style according to the documents I have found (Van Dale amongst others). As the `toDutch` function can only deterministically return a single value, I have chosen to go with the formal version. However, the `tryParseDutch` does accept the informal alternative (see marked line).

Comment (Mavamaarten) [inline Numsense.UnitTests/DutchExamples.fs]: I think the formal notation of `duizenddriehonderdzevenendertig` is more common, at least in Belgium. The formal style is the better choice here, imo.

**Pull request merged**